packages/shylu/shylu_dd/frosch/test/TestInterfaceSets/main.cpp

```diff
@@ -65,7 +65,11 @@ typedef unsigned UN;
 typedef double SC;
 typedef int LO;
 typedef int GO;
+#ifdef KOKKOS_HAVE_SERIAL
```

Comment:
Use `KOKKOS_ENABLE_SERIAL` not `KOKKOS_HAVE_SERIAL`.  The latter is deprecated and will be removed at the next Kokkos develop->master promotion.